modules/tinymce/src/core/main/ts/keyboard/Autocompleter.ts

```diff
@@ -99,25 +99,23 @@ export const setup = (editor: Editor): void => {
 
             // Ensure the active autocompleter trigger matches, as the old one may have closed
             // and a new one may have opened. If it doesn't match, then do nothing.
-            if (ac.trigger === context.trigger) {
-              // close if we haven't found any matches in the last 10 chars
-              if (context.text.length - ac.matchLength >= 10) {
-                cancelIfNecessary();
-              } else {
-                activeAutocompleter.set({
-                  ...ac,
-                  matchLength: context.text.length
-                });
-
-                if (uiActive.get()) {
-                  Events.fireAutocompleterUpdateActiveRange(editor, { range: context.range });
-                  Events.fireAutocompleterUpdate(editor, { lookupData });
-                } else {
-                  uiActive.set(true);
-                  Events.fireAutocompleterUpdateActiveRange(editor, { range: context.range });
-                  Events.fireAutocompleterStart(editor, { lookupData });
-                }
-              }
+            if (ac.trigger !== context.trigger) {
+              cancelIfNecessary();
```

Comment:
Is this really just removing the max length this seems to now call cancelIfNecessary when they are different before it only called this if the max length was to big. So should this just be a return or does it even matter?

Reply:
I removed it, so it's just a return now. I couldn't see a difference in testing

Reply:
The cancel function closes the autocompleter. If you look at the comment above this `if` block, when the `trigger` properties don't match then this autocompleter has closed and a new one opened. So yes it wasn't necessary.